src/driver/main.cpp

```diff
@@ -328,6 +328,7 @@ struct compiler
     compiler_target ct;
     bool offload_copy  = false;
     bool fast_math     = true;
+    bool tune          = false;
```

Comment:
I think it would be better to store `compiler_options` struct here, and then in `parse` just do `ap(co.offload_copy, ...)`(assuming it was named `co`).

Reply:
Done.